### PR TITLE
Ansible modules for PowerStore release version 2.0.0

### PR DESCRIPTION
**Pull request by Pavan-Mudunuri**
# Description

- Added support for manual appliance selection in volume module.
- Added support for replication group in replication session module.
- Info module is enhanced to list storage containers and replication group.
- Added new module for storage containers

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
|#23|
|#24|

# Checklist:

- [x] I have performed a self-review of my own code to ensure there are no formatting, pep8, linting, or security issues
- [x] I have performed Ansible Sanity test using --docker default
- [x] I have verified that new and existing unit tests pass locally with my changes
- [x] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Backward compatibility is not broken

# How Has This Been Tested?

- [x] Functional testing
- [x] Regression testing
- [x] Unit testing

### CODE REVIEW

File: plugins/doc_fragments/powerstore.py [hunk 1]
```diff
@@ -13,29 +13,30 @@ class ModuleDocFragment(object):
...
+              - C(true) - indicates that the SSL certificate should be
                        verified. Set the environment variable REQUESTS_CA_BUNDLE
                        to the path of the SSL certificate.
-              - False - indicates that the SSL certificate should not be
+              - C(false) - indicates that the SSL certificate should not be
                         verified.
           type: bool
-          required: True
-          choices: [True, False]
+          default: true
```
Comment (Bhavneet-Sharma) [inline plugins/doc_fragments/powerstore.py:27]: add **required: true** also

File: playbooks/modules/email.yml [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+        state: "present"
+
+    - name: Send a test mail to the destination email with email_id
+      dellemc.powerstore.email:
+        array_ip: "{{ array_ip }}"
+        user: "{{ user }}"
+        password: "{{ password }}"
+        validate_certs: "{{ validate_certs }}"
+        email_id: "xxx-xxxx-xxxx-xxx-xxxxxxx"
+        send_test_dellemc.powerstore.email: true
```
Comment (Bhavneet-Sharma) [inline playbooks/modules/email.yml:89]: it will be send_test_email

File: .ansible-lint [hunk 3]
```diff
@@ -0,0 +1,5 @@
+exclude_paths:
```
[the remark this answers is not in the document]
Comment (Pavan-Mudunuri) [reply .ansible-lint:1]: This file is required

File: CHANGELOG.rst [hunk 4]
```diff
@@ -5,6 +5,21 @@ Dellemc.Powerstore Change Logs
...
 
 
+v2.0.0
+======
+
+Minor Changes
+-------------
+
+- Added support for manual appliance selection in volume module.
+- Added support for replication group in replication session module.
```
[the remark this answers is not in the document]
Comment (Pavan-Mudunuri) [reply CHANGELOG.rst]: done

File: playbooks/modules/email.yml (hunk 2 above)
Comment (Pavan-Mudunuri) [reply playbooks/modules/email.yml]: done

File: .github/.ansible-lint [hunk 5]
```diff
@@ -0,0 +1,5 @@
+exclude_paths:
```
[the remark this answers is not in the document]
Comment (Pavan-Mudunuri) [reply .github/.ansible-lint]: removed

File: plugins/doc_fragments/powerstore.py (hunk 1 above)
Comment (Pavan-Mudunuri) [reply plugins/doc_fragments/powerstore.py:27]: not required

File: changelogs/changelog.yaml [hunk 6]
```diff
@@ -186,3 +186,14 @@ releases:
           name: vcenter
           namespace: ''
     release_date: '2023-03-31'
+  2.0.0:
+    changes:
+      minor_changes:
+        - Added support for manual appliance selection in volume module.
+        - Added support for replication group in replication session module.
```
[the remark this answers is not in the document]
Comment (Pavan-Mudunuri) [reply changelogs/changelog.yaml]: done

**Review by Bhavneet-Sharma — APPROVED** [review]

**Review by Jennifer-John — APPROVED** [review]

**Review by kuttattz — APPROVED** [review]

**Review by trisha-dell — APPROVED** [review]

**Pull request merged**